src/components/Collection/Collection.js

```diff
@@ -101,14 +131,16 @@ const Collection = ({ title, publications, text }) => (
 );
 
 Collection.propTypes = {
-  title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
-  publications: PropTypes.arrayOf(publicationType),
-  text: PropTypes.string,
+  title: oneOfType([string, element]).isRequired,
+  publications: arrayOf(publicationType),
+  text: string,
+  expanded: bool,
```

Comment:
The reason for this `expanded` prop is that the same component (`Collection`) is used on the home page and on the page for an individual collection. Instead of splitting it into two components, `expanded` is set to `true` when it is used to show an individual collection.

This works for now, but we should keep in mind that his component is used in two different situations. If there end up being additional differences in the future, we should split the component into two.